lib/util/ImpCentralApiHelper.js

```diff
@@ -454,6 +463,11 @@ class ImpCentralApiHelper {
         return error instanceof ImpCentralApi.Errors.ImpCentralApiError &&
             error._statusCode === 429;
     }
+
+    _isAuthenticationError(error) {
+        return error instanceof ImpCentralApi.Errors.ImpCentralApiError &&
+            error._statusCode === 401;
```

Comment:
or 403?

Reply:
According to the imp central spec and my experiments, 403 is used for insufficient permission error (eg. you can see a Devicegroup but are not permitted to modify it), not for invalid authorization.
It has no sense to refresh access token for it.

Reply:
fair enough